core/metrics/src/main/scala/org/typelevel/otel4s/metrics/ObservableInstrumentBuilder.scala

```diff
@@ -29,7 +29,7 @@ trait ObservableInstrumentBuilder[F[_], A, Instrument] {
     *   Unit</a>
     *
     * @param unit
-    *   the measurement unit. Must be 63 or fewer ASCII characters.
+    *   the measurement unit. Must be 255 or fewer ASCII characters.
     */
   def withUnit(unit: String): Self
```

Comment:
Hmm, I'm confused. The spec change is in https://github.com/open-telemetry/opentelemetry-specification/pull/3648 and it is about the instrument name, not the unit, which is documented below.

Reply:
Two pairs of eyes are better than one! 

Good point. I guess we can close the PR.